index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Martin Karugaba</title>
+    <link rel="stylesheet" href="./dist/output.css" />
+  </head>
+  <body class="font-poppins">
+    <!--* navbar -->
+    <header>
+      <nav
+        class="h-14 w-full px-5 py-4 bg-slate-50 flex justify-between items-center"
+      >
+        <!--* logo -->
+        <div
+          class="text-light-blue leading-3 gap-y-0 flex flex-col justify-between items-start cursor-pointer font-semibold"
+        >
+          <p class="text-2xl">My Logo</p>
+        </div>
+
+        <!--* nav links -->
+        <div
+          class="w-[26px] h-[30px] border-white flex flex-col justify-center items-center cursor-pointer"
+        >
+          <!--* hamburger -->
+          <div
+            class="h-[3px] w-[100%] bg-light-blue rounded-sm"
+          ></div>
+          <div
+            class="h-[3px] w-[100%] bg-light-blue my-[6px] rounded-sm"
+          ></div>
+          <div
+            class="h-[3px] w-[100%] bg-light-blue rounded-sm"
+          ></div>
+        </div>
+      </nav>
+    </header>
+    <!--* end navbar -->
+
+    <!--* header -->
+    <section
+      class="h-[600px] bg-hero-shapes bg-cover flex flex-col justify-center gap-y-4 items-start px-5"
+    >
+      <h1 class="text-dark-blue font-bold text-[40px] leading-11">
+        I'm Mariane <br />
+        Glad to see <br> you!
+      </h1>
+      <p class="text-base leading-7 text-light-teal">
+        I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and
+        experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+
+      <div>
+        <h2 class="uppercase text-4xl text-h2 font-medium">
+          Let's connect
+        </h2>
+
+        <!--* icons container-->
+        <div class="flex justify-between items-center gap-x-6 mt-2">
+          <!--* twitter icon -->
+          <a href="https://twitter.com/martin_karugaba">
+            <img
+              class="h-[20px] w-[20px] object-contain"
+              src="./img/twitter.png"
+              alt="twitter icon"
+            />
+          </a>
+
+          <!--* linkedin icon -->
+          <a
+            href="https://www.linkedin.com/in/martin-karugaba-822442173/"
```

Comment:
- [ ] like the previous reviewer noted, it is better to use an `ul` and `li `for the social icons. eg 

   
        <ul class="icons">
            <li>twitter</li>
            <li>linkedin</li>
            <li>etc</li>
          </ul>